test/watchCases/plugins/automatic-prefetch-plugin/1/foo/a.js

```diff
@@ -0,0 +1 @@
+module.exports = "This ";
```

Comment:
You don't have to copy equal files into each folder. Only add the diff to the step folders. As 0/foo/a.js is equal 1/foo/a.js you can omit 1/foo/a.js.